apps/log-collector/README.md

```
@@ -37,6 +37,7 @@ These are the environment variables you need to configure for production deploym
 | `LOG_MAX_FILE_SIZE_BYTES` | Max file size before rotation               | `10_485_760` (10MB) | `20_971_520` (20MB)                |
 | `LOG_MAX_ROTATED_FILES`   | Max number of rotated files                 | `5`                 | `10`                               |
 | `POD_LABEL_SELECTOR`      | Kubernetes label selector for pod discovery | unset (all pods)    | `"app=web,environment=production"` |
+| `POD_POLL_INTERVAL_MS`    | Interval between poll cycles (fallback mode) | `5000`              | `10000`                            |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)